webapp/actions/post_actions.jsx

```diff
@@ -30,7 +30,9 @@ export function handleNewPost(post, msg) {
         }
     }
 
-    var websocketMessageProps = null;
+    AsyncClient.getMyTeamMembersUnread();
```

Comment:
Adding another request every time the client receives a post might affect performance negatively. Maybe only do this if the post is not for the current team? Or possibly we can just fake it on the client (might be harder than it seems since we have to take into account someone viewing different teams on different devices)